frontend/src/components/mine/Compliance/MineComplianceInfo.js

```diff
@@ -24,90 +25,187 @@ const defaultProps = {
+                    <Col span={2} />
+                    <Col span={4}>
+                      <div className="center">
+                        <p className="info-display">
+                          {moment(this.props.mineComplianceInfo.last_inspection).format(
+                            "MMM DD YYYY"
+                          )}
+                        </p>
+                        <p className="info-display-label">LAST INSPECTION DATE</p>
+                      </div>
+                    </Col>
+                    <Col span={4} />
+                    <Col span={4}>
+                      <div className="center">
+                        <p className="info-display">
+                          {this.props.mineComplianceInfo.num_overdue_orders}
+                        </p>
+                        <p className="info-display-label">OVERDUE ORDERS</p>
+                      </div>
+                    </Col>
+                    <Col span={4} />
+                    <Col span={4}>
+                      <div className="center">
+                        <p className="info-display">
+                          {this.props.mineComplianceInfo.num_open_orders}
+                        </p>
+                        <p className="info-display-label">OPEN ORDERS</p>
+                      </div>
+                    </Col>
+                    <Col span={2} />
+                  </Row>
+                  <br />
+                  <br />
+                  <Row gutter={16} justify="center" align="top">
+                    <Col span={2} />
+                    <Col span={4}>
+                      <div className="center">
+                        <p className="info-display">{this.props.mineComplianceInfo.warnings}</p>
+                        <p className="info-display-label">WARNINGS ISSUED IN THE PAST YEAR</p>
+                      </div>
+                    </Col>
+                    <Col span={4} />
+                    <Col span={4}>
+                      <div className="center">
+                        <p className="info-display">{this.props.mineComplianceInfo.advisories}</p>
+                        <p className="info-display-label">ADVISORIES ISSUED IN THE PAST YEAR</p>
+                      </div>
+                    </Col>
+                    <Col span={4} />
+                    <Col span={4}>
+                      <div className="center">
+                        <p className="info-display">
+                          {this.props.mineComplianceInfo.section_35_orders} - Section 35
+                        </p>
+                        <p className="info-display-label">TOTAL</p>
+                      </div>
+                    </Col>
+                    <Col span={2} />
+                  </Row>
+                </div>
+              )}
+              {!this.props.mineComplianceInfo && <NullScreen type="generic" />}
+            </div>
+          )}
+        </div>
+        <br />
+        <br />
+        <div>
```

Comment:
This really seems like 2 different components and/or concerns to me. One is a compliance overview, and one is open orders. I'd love for @mmumby to weigh in on this.

If we don't split them into separate files, I think it would at least be a good idea to split out the JSX into methods or `const`s to make this render method more manageable.

Reply:
Pulled this out into a separate function.